modules/loader/src/main/java/com/opengamma/strata/loader/csv/PositionCsvInfoResolver.java

```diff
@@ -114,19 +133,109 @@ public default SecurityPosition completePosition(CsvRow row, SecurityPosition po
     return position;
   }
 
+  //-------------------------------------------------------------------------
   /**
    * Parses the contract specification from the row.
    * 
    * @param row  the CSV row to parse
    * @param type  the ETD type
-   * @return the loaded positions, position-level errors are captured in the result
+   * @return the contract specification
+   * @throws IllegalArgumentException if the specification is not found
    */
   public default EtdContractSpec parseEtdContractSpec(CsvRow row, EtdType type) {
-    ExchangeId exchangeId = ExchangeId.of(row.getValue(StandardCsvInfoResolver.EXCHANGE_FIELD));
-    EtdContractCode contractCode = EtdContractCode.of(row.getValue(StandardCsvInfoResolver.CONTRACT_CODE_FIELD));
+    ExchangeId exchangeId = ExchangeId.of(row.getValue(EXCHANGE_FIELD));
+    EtdContractCode contractCode = EtdContractCode.of(row.getValue(CONTRACT_CODE_FIELD));
     EtdContractSpecId specId = EtdIdUtils.contractSpecId(type, exchangeId, contractCode);
     return getReferenceData().findValue(specId).orElseThrow(
         () -> new IllegalArgumentException("ETD contract specification not found in reference data: " + specId));
   }
 
+  /**
+   * Parses an ETD future position from the CSV row.
+   * <p>
+   * In most cases this will return {@link EtdFuturePosition}, however for flexibility
+   * it is allowed to return any position, such as {@link SecurityPosition}.
```

Comment:
The comment doesn't match the return type. Should it actually return `Position` like the method below?